### ISSUE TITLE
Pictures automatic upload

### ISSUE THREAD

**Issue by xplosionmind** [Piwigo/Piwigo-Mobile#439]
**Is your feature request related to a problem? Please describe.**   
To me, bing able to upload photos only after having selected them is not very useful. If pictures could be automatically uploaded, too, it would be great and super useful.

**Describe the solution you'd like**   
Selecting specific albums and/or folders of pictures and automatically upload them to a specific Piwigo album

**Describe alternatives you've considered**   
Having to go to the Piwigo app every time and uploading media manually

**Additional context**   
It would be great, as I wrote before, if a specific album could be selected, and also provide the possibility of adding e.g. the `shotoniphone` tag to pictures automatically uploaded.

**Comment by EddyLB** [comment]
Duplicate of #70

**Comment by xplosionmind** [comment]
Sorry, I did not notice
